0010-vrf-elections.md

```diff
@@ -36,7 +36,7 @@ following additions and extra clarifications:
   enforcing this when decoding.
 
 - When decoding s in the ECVRF_verify routine, the s scalar MUST fall
-  within the range 0 <= i < L.  This change will make proofs
+  within the range 0 ≤ i < L.  This change will make proofs
```

Comment:
```suggestion
  within the range 0 &le; i < L.  This change will make proofs
```
Please use the HTML entity for those of us whose keyboards do not fit the entire UTF-8 charset.